change-beta/@azure-communication-react-c4bb9de6-fe63-4936-bf91-f6e57a73e6a8.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "[ChatSDK] Version bump",
```

Mention the exact version it is bumped to so anyone reading the changelog can easily know what to expect
```suggestion
  "comment": "`@azure/communication-chat` peer dependency updated to 1.3.2-beta.2 for beta packages",
```